exchange/adapter_builders.go

```diff
@@ -329,6 +329,7 @@ func newAdapterBuilders() map[openrtb_ext.BidderName]adapters.Builder {
 		openrtb_ext.BidderVideoHeroes:       videoheroes.Builder,
 		openrtb_ext.BidderVidoomy:           vidoomy.Builder,
 		openrtb_ext.BidderViewdeos:          adtelligent.Builder,
+		openrtb_ext.BidderUndertone:         undertone.Builder,
```

Comment:
preference is to list in ascending order. Should move this line after `openrtb_ext.BidderUcfunnel:          ucfunnel.Builder`


```
...
openrtb_ext.BidderUcfunnel:          ucfunnel.Builder,
openrtb_ext.BidderUndertone:         undertone.Builder,
openrtb_ext.BidderUnicorn:           unicorn.Builder,		

...
```